completion/fish/task.fish

```diff
@@ -27,3 +35,4 @@ complete -c $GO_TASK_PROGNAME -s t -l taskfile  -d 'choose which Taskfile to run
 complete -c $GO_TASK_PROGNAME -s v -l verbose   -d 'enables verbose mode'
 complete -c $GO_TASK_PROGNAME      -l version   -d 'show Task version'
 complete -c $GO_TASK_PROGNAME -s w -l watch     -d 'enables watch of the given task'
+
```

Comment:
This extra empty line here is not needed. (Should have a single empty line, we now have two).

Reply:
Hmm, previously there were no new line at the end, so I added it, right now it looks like this:
![image](https://user-images.githubusercontent.com/39216424/195945595-ddf96636-523b-40fb-877c-2f83e8ffaf95.png)

Reply:
Removed that to match all other files